tests/services/repo-creator.service.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { RepoCreatorService } from '../../src/app/core/services/repo-creator.service';
+import { of } from 'rxjs';
+
+const PHASE_OWNER = 'CATcher-org';
+const PHASE_REPO = 'bugreporting';
+let repoCreatorService: RepoCreatorService;
+let githubService: any;
+
+describe('RepoCreatorService', () => {
+  beforeEach(() => {
+    githubService = jasmine.createSpyObj('GithubService', ['isRepositoryPresent']);
+    repoCreatorService = new RepoCreatorService(githubService);
+  });
+
+  describe('.verifyRepoCreation)_', () => {
```

Comment:
I think there's a typo here! Should be `verifyRepoCreation()` right?😆

Reply:
ah yes typo 😅

Reply:
Fixed 👍